### PR TITLE
PXP-650: [CLI] Develop a panel on the TUI to display AppSignal data

### PR DESCRIPTION
**Pull request by jk-gan**
## Summary

Screenshot 2024-02-20 at 3 16 41 PM

This PR change the log panel in TUI to a Tab Widget. The first tab is showing GCloud logs and the second tab is showing Appsignal data. To switch tab, you need to select the log panel first, then press `1` to select the first tab and press `2` to select the second tab. I wanted to use `ctrl+1` and `ctrl+2` instead but there is a bug from the keyboard event.

The appsignal data will be refetched every 10 seconds.

Btw, I also change the timeframe for the error rates and throughputs. In the [RFC](https://mindvalley.atlassian.net/wiki/spaces/PXP/pages/762773701/RFC19+-+Display+AppSignal+Data+On+The+Wukong+CLI), the error rates and the throughputs should show **1h**, **6h** and **24h** average values. However, there is no **6h** timeframe from the appsignal API, so I change it to **8h** instead. cc @onimsha 

Ticket: [PXP-650](https://mindvalley.atlassian.net/browse/PXP-650)

### Added
- [x] Add a new panel to show Appsignal data including Error rates, Throughputs and Latencies.
- [x] Add a way to read the local application config file `.wukong.toml`
- [x] Add a way to initialise Appsignal integration in `wukong application init`

### Changed
- [x] More error handling for `ApplicationConfigs`
- [x] The Log panel in TUI is now Tab, for now it will display GCloud log on the first tab and Appsignal data on the second tab.
- [x] Refactor some codes and move them to the proper forlder.
- [x] Start fetching API during the welcome screen, this will speed up the TUI by 3 seconds+ 

[PXP-650]: https://mindvalley.atlassian.net/browse/PXP-650?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/196#" title="Author self-approved">jk-gan</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by onimsha — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/wukong-cli/pull/196#" title="Author self-approved">jk-gan</a>*, *<a href="https://github.com/mindvalley/wukong-cli/pull/196#pullrequestreview-1897279975" title="Approved">onimsha</a>*


The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fwukong-cli).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/wukong-cli/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
